rdm/pull.py

```diff
@@ -5,20 +5,16 @@
 '''
 import sys
 
-from rdm.util import load_yaml, write_yaml, print_info
+from rdm.util import load_yaml, write_yaml, print_info, load_class
 
 
-def pull_from_project_manager(system_yml_path, cache_dir=None):
-    system = load_yaml(system_yml_path)
+def pull_from_project_manager(config_path):
+    config = load_yaml(config_path)
 
-    if system['project_management_tool'] == 'GitHub Issue':
-        from rdm.backends.github_issue import pull
-    elif system['project_management_tool'] == 'GitHub PR':
-        from rdm.backends.github_pr import pull
-    else:
-        raise ValueError("Project management tool not supported.")
+    BackendClass = load_class(config['project_management_backend'])
+    pm_backend = BackendClass(config)
 
-    development_history = pull(system, cache_dir)
+    development_history = pm_backend.pull()
```

Comment:
Awesome. I think the refactors done here will be a really big usability improvement!